owncloudApp/src/main/java/com/owncloud/android/presentation/ui/security/passcode/SquareFrameLayout.kt

```diff
@@ -0,0 +1,42 @@
+package com.owncloud.android.presentation.ui.security.passcode
```

Comment:
You forgot to add the copyright to the file! Don't worry, ask us how to configure it and we will help you 😉